src/codegen/xrccg.cpp

```diff
@@ -46,7 +46,7 @@ bool XrcCodeGenerator::GenerateCode( PObjectBase project )
 	doc.LinkEndChild( &decl );
 
 	ticpp::Element element( "resource" );
-	element.SetAttribute( "xmlns", "http://www.wxwindows.org/wxxrc" );
+	element.SetAttribute( "xmlns", "http://www.wxwidgets.org/wxxrc" );
```

Comment:
New code shouldn't use that whitespace padding of parentheses you can find frequently in the code base, can you remove that please?